src/Files.App/Actions/Content/Archives/CompressIntoArchiveAction.cs

```diff
@@ -1,25 +1,22 @@
 ﻿// Copyright (c) 2023 Files Community
 // Licensed under the MIT License. See the LICENSE.
 
-using CommunityToolkit.Mvvm.DependencyInjection;
 using Files.App.Contexts;
 using Files.App.Dialogs;
-using Files.App.Extensions;
 using Files.App.Filesystem.Archive;
-using Files.App.Helpers;
 using Microsoft.UI.Xaml.Controls;
-using System.ComponentModel;
-using System.Threading.Tasks;
 
 namespace Files.App.Actions
 {
 	internal class CompressIntoArchiveAction : BaseUIAction, IAction
 	{
-		private readonly IContentPageContext context = Ioc.Default.GetRequiredService<IContentPageContext>();
+		private readonly IContentPageContext context;
 
-		public string Label => "CreateArchive".GetLocalizedResource();
+		public string Label
+			=> "CreateArchive".GetLocalizedResource();
```

Comment:
I remember there was some kind of issue with the rendering using arrow instead of `{ get; }`. I'll let you know